api/report_server.thrift

```diff
@@ -298,6 +305,30 @@ service codeCheckerDBAccess {
                                              4: ReportFilterList reportFilters)
                                              throws (1: shared.RequestFailed requestError),
 
+  // The Counter value should be converted beck to the
```

Comment:
Please mark the outdated API function  
`ReportDataTypeCountList getRunResultTypes(1: i64 runId, 2: ReportFilterList reportFilters)
                                            throws (1: shared.RequestFailed requestError)`
outdated in the API description and place a TODO so that we do not forget to remove this.